v2.0/admin-ui-node-map.md

```diff
@@ -0,0 +1,185 @@
+
+## Why Use the Node Map View
+
+The **Node Map View** helps you monitor the cluster health and performance, ensure effective resource utilization, and troubleshoot cluster issues:
+
+### Monitor Cluster Health and Performance
+
+The **Node Map** view provides:
+
+- A high-level graphical overview of how the cluster is configured, which helps you understand how the available hardware is servicing your users. 
+- Important cluster health metrics, such as Capacity and CPU utilization, QPS, and uptime.
+
+### Ensure Effective Resource Utilization
+
+The **Node Map** view helps you:
+
+- Confirm that the database is balanced and you are utilizing the resource effectively. This helps you provision hardware and plan ahead for scale.
+- Ensure that storage capacity is balanced across the disks available to the extent that they match the zone configs.
+- Check if there resource congestion anywhere (for example certain nodes going beyond capacity).
+
+### Troubleshoot the Cluster
+
+The **Node Map** view helps you identify and preempt any scenarios that might cause downtime, service issues, or otherwise negatively impact cluster health. The **Node Map** view helps you:
+
+- Monitor disk utilization to know if you need to add more storage capacity.
+- Identify dead nodes in the cluster.
+- Be informed about unavailable or under-replicated ranges.
+- Confirm that configuration changes were persisted to the cluster.
+- Observe QPS across nodes to ensure that queries are rerouted to live nodes if a node failure occurs.
+
+## Configure and Navigate the Node Map
+
+To configure the **Node Map**, you need to start the cluster with the correct `--localities` flags and assign the latitudes and longitudes for each region.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> won't be displayed until all regions are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+
+Consider a four-node geo-distributed cluster with the following configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  N1 | us-east | us-east-1 |
+|  N2 | us-east | us-east-1 |
+|  N3 | us-west | us-west-1 |
+|  N4 | eu-west | eu-west-1 |
+
+#### Step 1. Ensure the CockroachDB version is 2.0 or higher
+
+~~~ shell
+$ ./cockroach version
+~~~
+
+If not, [upgrade to CockroachDB v2.0](upgrade-cockroach-version.html).
+
+#### Step 2. Start the nodes with the correct `--localities` flags
+
+In a new terminal, start Node 1:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=us-east,datacenter=us-east-1  \
+--store=node1 \
+--host=localhost \
+--port=26257 \
+--http-port=8080 \
+--join=localhost:26257,localhost:26258,localhost:26259
```

Comment:
nit: only need to specify one other address in join flag. Also, the first node doesn't have anyone to join, so maybe leave it off.

Reply:
I kept this consistent with other docs.